roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -2,17 +2,14 @@
 from openshift_checks import get_var
 
 
-class NotContainerized(object):
+class NotContainerizedMixIn(object):
```

Comment:
@rhcarvalho It looks like [The Hitchhiker's Guide to Python](https://books.google.com/books?id=4yXtDAAAQBAJ&lpg=PT218&ots=XuaM-p9fXR&dq=python%20mixin%20naming%20convention&pg=PT218#v=onepage&q=python%20mixin%20naming%20convention&f=false) is on @ashcrow's side here for naming.